sources/packages/web/src/components/generic/HeaderNavigator.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <slot name="title">
+    <div v-if="!routeName" class="form-header-title">{{ title }}</div>
+    <div v-if="routeName" class="form-header-title">
+      <a @click="goBack(routeName, routeParams)">
+        <v-icon left> mdi-arrow-left </v-icon> {{ title }}</a
```

Comment:
For icons, we have decided to use font-awesome-icons slowly replacing all the material icons with font-awesome. @ann-aot . Am I missing anything ?

Reply:
yes, we need to replace the icons with `font-awesome-icons`. maybe we can create a technical debt

Reply:
In this case, it is a new component, so we should go with font-awesome while building it.